rfc/README.md

```diff
@@ -6,6 +6,9 @@ We write an RFC to _communicate intent and get feedback_ before we start buildin
 
 Whenever your team is making a decision that could affect the platform, write the details and get feedback from others on the Platform. Anyone - including product managers, UX, research, and engineers - is encouraged to create RFCs in order to solicit ideas in preparation of a decision direction.
 
+There is no formal requirement to present an RFC at the [Platform Architecture Working Group meeting](https://vfs.atlassian.net/wiki/spaces/ECP/pages/1872363553/Platform+Architecture+Working+Group), but authors may _request_ to present there. Depending on the subject and/or scope of an RFC the Platform Architecture WG _may_ request author(s) to present either by its own authority or because it has been petitioned to do so.
+Requesting an RFC's presentation is as simple as adding the RFC's PR to [the docket of the next scheduled meeting](https://vfs.atlassian.net/wiki/spaces/ECP/pages/2160132128/Working+Group+Meeting+Notes) along with a brief explanation of what the desired outcome of presentating is.
```

Comment:
I thought this was currently handled by automation upon creating of the pull request. Did I misunderstand something?

Reply:
Getting on the Docket of the Platform Architecture Working Group is different than getting on the GitHub Board.

The former is just getting on the schedule to present and does not have automation. That's just editing a Confluence page or requesting it to be edited.
There **is** automation to get on the latter (https://github.com/orgs/department-of-veterans-affairs/projects/710)

Now that being said, I will add some additional information to clarify that one could also request getting on the docket via Slack because a VFS may not necessarily have access to edit that.